bin/compile

```diff
@@ -237,3 +239,8 @@ if [ -n "${BUILDPACK_CLEAR_CACHE+1}" ]; then
   echo "-----> Clearing cache dir."
   rm -rf $METEOR_DIR
 fi
+
+# Calculate execution time
+END=$(date +%s)
+DIFF=$(($END - $START))
+echo "-----> Build duration: $(date -d@$DIFF -u +%H:%M:%S)"
```

Comment:
Thanks for the PR!

Could you add a note saying this date call doesn't work on macOS. Using BSD, it would be `$(date -r $DIFF -u +%H:%M:%S)`

Reply:
Or you can be fancy and [use uname for OS detection](https://stackoverflow.com/questions/394230/how-to-detect-the-os-from-a-bash-script/18434831) and pick the right arguments.

I'll also point out that this won't work for times longer than a day (since it'll only do hours up to 24), but I really hope that never comes up.

If you really want to solve both these problems, you could always do
```bash
printf "%d:%02d:%02d\n" $((DIFF/3600)) $(($DIFF/60%60)) $((DIFF%60))
```
But that's probably silly.